### PR TITLE
Onboarding instructions for adding AWS account

### PR DESCRIPTION
**Pull request by bess**
Connected to https://github.com/pulibrary/pdc_discovery/issues/303

### CODE REVIEW

File: onboarding.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+
+
+## AWS Setup for S3 access
+RDSS maintains S3 buckets on AWS for storing research data. Here are the necessary steps to set up a new user with the correct permissions to help maintain these. 
+
+
+### 1. Set up AWS account
+1. Go to [https://princeton.edu/aws](https://princeton.edu/aws) and log in with Princeton credentials.
+2. From the "Services" menu, select "IAM". Create a new user, with AWS credential type "Access Key". Add the new account to the group `rdss`.
```
Comment (carolyncole) [inline onboarding.md]: I had to search for IAM in the search bar.  It was not part of my services.

Comment (carolyncole) [reply onboarding.md]: I then had to click users on the left.

Comment (carolyncole) [reply onboarding.md]: I there a user name I should specify? I guessed and put it in as my netid

File: onboarding.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+
+### 1. Set up AWS account
+1. Go to [https://princeton.edu/aws](https://princeton.edu/aws) and log in with Princeton credentials.
+2. From the "Services" menu, select "IAM". Create a new user, with AWS credential type "Access Key". Add the new account to the group `rdss`. 
+3. Record the Access Key ID and the Secret access key you'll get on the `Success` screen. You will need to add these to your local user profile when you set up AWS command line access. 
+
+### 2. Install and configure aws cli
+1. Install the aws cli: https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html
+2. Configure it using the Access key ID and Secret Access key attached to your account
```
Comment (carolyncole) [inline onboarding.md]: Should link to: https://docs.aws.amazon.com/cli/latest/userguide/getting-started-quickstart.html

Comment (carolyncole) [reply onboarding.md]: Is there a default region we should pick?

Comment (carolyncole) [reply onboarding.md]: Could also just put in the step `aws configure`

File: onboarding.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+3. You will also need to install jq: `brew install jq` (assuming you use homebrew)
+
+### 3. Test whether it worked
+You should be able to issue this command:
+```
+aws ec2 describe-instances --query 'Reservations[*].Instances[?not_null(PublicIpAddress)]' | jq -r '.[][]|.PublicIpAddress+" "+(.Tags[]|select(.Key=="Name").Value)'
```
Comment (carolyncole) [inline onboarding.md:30]: I get this error 
`You must specify a region. You can also configure your region by running "aws configure".`

**Review by carolyncole — CHANGES_REQUESTED** [review]
@bess I went through the instructions and commented on my questions. Things were a bit vague.  I guessed best I could...  I'm going to guess on a region and see if that gets me past the error.

**Comment by carolyncole** [comment]
After reading the aws documentation I chose a region of `us-east-1` and a type of `json` and the test command did give me some IP addresses

**Review by jrgriffiniii — APPROVED** [review]

**Comment by jrgriffiniii** [comment]
Thank you very much for this, running the Playbook (as specified in https://github.com/pulibrary/princeton_ansible/pull/3080/files) confirmed that this was indeed adding me to the `rdss` group.

**Review by jrgriffiniii — APPROVED** [review]

**Review by carolyncole — APPROVED** [review]

File: onboarding.md [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+
+### 1. Set up AWS account
+1. Go to [https://princeton.edu/aws](https://princeton.edu/aws) and log in with Princeton credentials.
+2. From the "Services" menu, select "IAM". (You might need to search for IAM in the search bar.) Under the "User" menu, create a new user. Use any user name you prefer, as long as it would be clear to your team mates that this name is associated with you. Choose AWS credential type "Access Key". Add the new account to the group `rdss`. 
+3. Record the Access Key ID and the Secret access key you'll get on the `Success` screen. You will need to add these to your local user profile when you set up AWS command line access. 
+
+### 2. Install and configure aws cli
+1. Install the aws cli: https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html
+2. Configure it using the Access key ID and Secret Access key attached to your account. Do this by running `aws configure` or following instructions [here](https://docs.aws.amazon.com/cli/latest/userguide/getting-started-quickstart.html). 
+  * Use default region `us-east-1`
```
Comment (jrgriffiniii) [inline onboarding.md:23]: 👍

**Review by jrgriffiniii — APPROVED** [review]

**Pull request merged**